lib/katello/tasks/pulp3_content_switchover.rake

```diff
@@ -3,13 +3,15 @@ require "#{Katello::Engine.root}/app/services/katello/pulp3/migration_switchover
 
 namespace :katello do
   desc "Runs a Pulp 3 migration of pulp3 hrefs to pulp ids for supported content types."
-  task :pulp3_content_switchover => :environment do
+  task :pulp3_content_switchover => ["dynflow:client"] do
+    dryrun = ENV['DRYRUN']
     begin
       User.current = User.anonymous_admin
 
       ActiveRecord::Base.transaction do
         switchover_service = Katello::Pulp3::MigrationSwitchover.new(SmartProxy.pulp_primary)
         switchover_service.run
+        fail "Dryrun completed without error, aborting and rolling back" if dryrun
       end
     rescue Katello::Pulp3::SwitchOverError => e
       $stderr.print(e.message)
```

Comment:
Might wanna add helpful description like here like "Associated Stack trace (not to worry)" types because the switchover command caused the following output
```
[root@centos7-katello-3-18 katello]# foreman-rake katello:approve_corrupted_migration_content
enabled
Any missing or corrupt content will be ignored on migration to Pulp 3.  This can be undone with 'foreman-rake katello:pulp3_migration_unapprove_corrupted'
[root@centos7-katello-3-18 katello]# foreman-rake katello:pulp3_content_switchover DRYRUN=true
rake aborted!
Dryrun completed without error, aborting and rolling back
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.18.1/lib/katello/tasks/pulp3_content_switchover.rake:14:in `block (3 levels) in <top (required)>'
/opt/theforeman/tfm/root/usr/share/gems/gems/activerecord-6.0.3.4/lib/active_record/connection_adapters/abstract/database_statements.rb:280:in `block in transaction'
/opt/theforeman/tfm/root/usr/share/gems/gems/activerecord-6.0.3.4/lib/active_record/connection_adapters/abstract/transaction.rb:280:in `block in within_new_transaction'
/opt/theforeman/tfm/root/usr/share/gems/gems/activesupport-6.0.3.4/lib/active_support/concurrency/load_interlock_aware_monitor.rb:26:in `block (2 levels) in synchronize'
/opt/theforeman/tfm/root/usr/share/gems/gems/activesupport-6.0.3.4/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `handle_interrupt'
/opt/theforeman/tfm/root/usr/share/gems/gems/activesupport-6.0.3.4/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `block in synchronize'
/opt/theforeman/tfm/root/usr/share/gems/gems/activesupport-6.0.3.4/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `handle_interrupt'
/opt/theforeman/tfm/root/usr/share/gems/gems/activesupport-6.0.3.4/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `synchronize'
/opt/theforeman/tfm/root/usr/share/gems/gems/activerecord-6.0.3.4/lib/active_record/connection_adapters/abstract/transaction.rb:278:in `within_new_transaction'
/opt/theforeman/tfm/root/usr/share/gems/gems/activerecord-6.0.3.4/lib/active_record/connection_adapters/abstract/database_statements.rb:280:in `transaction'
/opt/theforeman/tfm/root/usr/share/gems/gems/activerecord-6.0.3.4/lib/active_record/transactions.rb:212:in `transaction'
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.18.1/lib/katello/tasks/pulp3_content_switchover.rake:11:in `block (2 levels) in <top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/rake-12.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => katello:pulp3_content_switchover
(See full trace by running task with --trace)

```
It says `Dryrun completed without error, aborting and rolling back` before dumping the stack trace. But user might not realize its ok.